app/(golems)/golems/components/Sidebar.tsx

```
@@ -0,0 +1,174 @@
+              <span className="font-bold text-[#e59500] text-sm tracking-tight">Golems</span>
+            </Link>
+          </div>
+        </header>
+      )}
+
+      {/* Mobile overlay */}
+      {mobileOpen && (
+        <div
+          className="fixed inset-0 bg-black/60 z-40 md:hidden"
+          role="button"
+          tabIndex={0}
```

Biome (2.3.14)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)